Example/Example-Objective-C Tests/DummyTest.m

```diff
@@ -0,0 +1,31 @@
+#import <XCTest/XCTest.h>
```

Comment:
The dummy tests are just to ensure the Examples build and run properly. The swift test dummy will verify that Objective-C notifications for example bridges to Swift and the Objective-C dummy test will verify that the rest of the library bridges to Objective-C.